Components/Metrics/AdvancedMattesMutualInformation/itkParzenWindowMutualInformationImageToImageMetric.hxx

```diff
@@ -555,42 +555,12 @@ ParzenWindowMutualInformationImageToImageMetric<TFixedImage, TMovingImage>::Afte
   const ThreadIdType numberOfThreads = Self::GetNumberOfWorkUnits();
 
   /** Accumulate derivatives. */
-  // compute single-threadedly
-  if (!Superclass::m_UseMultiThread && false) // force multi-threaded
```

Comment:
This `if (!x && false)` was introduced with commit 2fc2c495837f64416263fa7c5d1e2bc4e3e061fa, "HUGE merge of the performance branch", November 14, 2013